CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
+<<<<<<< HEAD
```

Comment:
looks like merge error?

Reply:
Fixed. Forgot to save after resolving the conflict.